diff --git a/models/staging/stg_bad_orders.sql b/models/staging/stg_bad_orders.sql
new file mode 100644
--- /dev/null
+++ b/models/staging/stg_bad_orders.sql
@@ -0,0 +1,8 @@
+-- A deliberately bad staging model for demo purposes
+
+SELECT *
+FROM raw.jaffle_shop.orders
+LEFT JOIN raw.jaffle_shop.payments p
+    ON orders.id = p.order_id
+WHERE status = 'completed'
+ORDER BY orders.order_date DESC
